plugins/module_utils/_vmware.py

```diff
@@ -262,25 +262,31 @@ def get_objs_by_name_or_moid(self, vimtype, name, return_all=False, search_root_
             # for backwards-compat
             return None
 
-    def get_standard_portgroup(self, portgroup):
+    def get_dvs_portgroup_by_name_or_moid(self, identifier, fail_on_missing=False):
```

Comment:
I think this is wrong name of the method. shouldn't be `get_standard_portgroup_by_name_or_moid`?

Reply:
ahh good catch. interesting that none of the tests failed since its a duplicate method name....